pppp/src/App.vue

```diff
@@ -1,15 +1,24 @@
 <template>
-  <div id="app">
-    <Header :title='pageTitle'
-            imagePath='/pppp/images/' />
-    <main>
-      <div class="container stepper">
-        <ProgressBar :currentPath='$router.currentRoute.path'
-                    :routes='stepRoutes'/>
-      </div>
-      <router-view/>
-    </main>
-    <Footer :version='version' />
+  <div>
+    <div :aria-hidden="[isConsentModalOpen]">
+      <Header :title='pageTitle'
+              imagePath='/pppp/images/' />
+      <main>
```

Comment:
I think Ernesto usually wants the nav tags before and not inside main tags. So the main tags might be better wrapping right around the router view and excluding the page stepper